### ISSUE TITLE
Runtime Error When Scrolling Notifications

### ISSUE THREAD

**Issue by MickTheLinuxGeek** [RasmusLindroth/tut#104]
Good day.

I'm having an issue while using tut. I'm using POP! OS as my operating system and has all updates applied. I've tried the amd64 binary and also building from the source with the same issue. What happens: I navigate down to the notifications section and begin to scroll through them. I get to the 10th notification and try to go the the 11th, the app crashes every time with a runtime error. I've attached a file that contains the stack dump messages. If there is any other info you need, please let me know.

Thanks,

/Mike Biel
mick.the.linux.geek@hotmail.com

![Screenshot from 2021-11-17 16-03-42](https://user-images.githubusercontent.com/32104784/142285678-ca7547b4-d283-4f3a-a04e-3d6252c34c1d.png)

**Comment by RasmusLindroth** [comment]
That's weird, the status seems to be empty for some reason. What instance are you on? And could you check what type of notification it is in the browser? So just scroll through your notifications in the browser and see which type of notification that makes tut crash. 

The fix will be easy, but I want to know if I've missed to handle some possible notification, so tut can handle it correctly.

Thank you for the bug report :)

**Comment by MickTheLinuxGeek** [comment]
Hey! Sorry it has taken me awhile to check this and get back you. I've updated to the newest version and it seems to be working fine. I am running on the mastodon.online instance. Since I filed the issue I have deleted all my mastodon notifications, so what I have done is probably not a true test, but I'll keep a look out. It is a cool CLI app. Thanks for fixing the issue.

/Mike Biel - mick.the.linux.geek@hotmail.com
